pkg/investigations/chgm/chgm.go

```diff
@@ -82,9 +82,11 @@ func Investigate(r *investigation.Resources) error {
 		logging.Infof("Network verifier reported failure: %s", failureReason)
 
 		if strings.Contains(failureReason, "nosnch.in") {
-			err := r.OcmClient.PostLimitedSupportReason(createEgressLS(failureReason), r.Cluster.ID())
+			err := r.OcmClient.PostLimitedSupportReason(createEgressLS(failureReason), r.Cluster)
 			if err != nil {
-				return err
+				notes.AppendWarning("NetworkVerifier found unreachable targets, deadmanssnitch is blocked! \nUnreachable: \n%s", failureReason)
```

Comment:
Nice catch. Did mindlessly copy this.
I think, it might be best to just remove the mention of DMS as its not relevant to the workflow here.
would look like this, wdyt?

🤖 Automated CHGM pre-investigation 🤖
===========================
✅ Customer did not stop nodes.
⚠️ NetworkVerifier found unreachable targets!
Unreachable: 
nosnch.in:443
⚠️Cannot post limited support on critical customers cluster. Please open a proactive case and silence the alert.